<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveLobbyWidget</class>
 <widget class="QWidget" name="LiveLobbyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LiveLobbyWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QLineEdit" name="m_searchLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #000000;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_searchPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8fc0e7;</string>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_openLivePushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8fc0e7;</string>
   </property>
   <property name="text">
    <string>开播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_settingPushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8fc0e7;</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="m_scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>750</width>
     <height>480</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
